.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+.idea
```

Comment:
This should probably go in your system .gitignore

Reply:
makes sense!